implementations/java-json-schema/BowtieJsonSchema.java

```diff
@@ -32,6 +34,30 @@ public class BowtieJsonSchema {
   private final PrintStream output;
   private boolean started;
 
+  static {
+    MANIFEST_ATTRIBUTES = readManifestAttributes();
+    UNSUPPORTED =
+        "1.7.0".compareTo(
```

Comment:
Just use the first map:
```java 
Map.of("$ref prevents a sibling $id from changing the base uri",
                     RECOGNIZING_IDENTIFIERS)
```

The second one is taking into account an older version of test suite so it isn't really related to my impl version